test/web-vitals-instrumentation.test.ts

```diff
@@ -0,0 +1,252 @@
+import { WebVitalsInstrumentation } from '../src/web-vitals-autoinstrumentation';
+import {
+  BasicTracerProvider,
+  InMemorySpanExporter,
+  SimpleSpanProcessor,
+} from '@opentelemetry/sdk-trace-base';
+
+describe('Web Vitals Instrumentation Tests', () => {
```

Comment:
For tests, decided to focus on the spans being generated instead of whether the web vitals functions fire when they're supposed to since that's covered in the tests for the web-vitals library. Will eventually want to write smoke tests for this but will make that a separate PR.